Homework2/HomeworkAnswers.txt

```diff
@@ -0,0 +1,11 @@
+Experiment 2:
+
+The Dice coefficient has gone down to 0.0740741. 
+This is even worse overlap than we saw in Experiment 1.
+Because we removed motion censoring and putting motion coefficients into the regression,
+motion is less accounted for. Thus, the same voxel in space may correspond to different brain regions.
```

Comment:
In both cases, we have done motion correction, we are just changing whether information about how much the subject moved is included in the regression or not. So the voxels are in the same location in both cases, but we learn from this exercise that there are still motion-induced artifacts in the data that can greatly affect the reliability of the results.